src/actions/ansible-repository-collection-version-add.tsx

```diff
@@ -8,29 +8,43 @@ import {
 } from 'src/api';
 import { AlertList, AlertType, DetailList, closeAlert } from 'src/components';
 import { canEditAnsibleRepository } from 'src/permissions';
-import { handleHttpError, parsePulpIDFromURL, taskAlert } from 'src/utilities';
+import {
+  RepositoriesUtils,
+  handleHttpError,
+  parsePulpIDFromURL,
+  taskAlert,
+} from 'src/utilities';
 import { Action } from './action';
 
 const add = (
   { repositoryHref, repositoryName },
-  { namespace, name, version, pulp_href: collectionVersionHref },
+  collections,
   { addAlert, setState, query },
 ) => {
   const pulpId = parsePulpIDFromURL(repositoryHref);
-  return AnsibleRepositoryAPI.addContent(pulpId, collectionVersionHref)
+  const collectionVersionHrefs = collections.map(
+    (c) => c.collection_version.pulp_href,
+  );
+  return AnsibleRepositoryAPI.addContent(pulpId, collectionVersionHrefs)
     .then(({ data }) => {
-      addAlert(
-        taskAlert(
-          data.task,
-          t`Started adding ${namespace}.${name} v${version} to repository "${repositoryName}".`,
-        ),
+      collections.map(
+        ({ collection_version: { name, namespace, version } }) => {
+          addAlert(
+            taskAlert(
+              data.task,
+              t`Started adding ${namespace}.${name} v${version} to repository "${repositoryName}".`,
+            ),
+          );
+          setState((ms) => ({ ...ms, addCollectionVersionModal: null }));
+          query({});
+        },
       );
-      setState((ms) => ({ ...ms, addCollectionVersionModal: null }));
-      query({});
     })
     .catch(
       handleHttpError(
-        t`Failed to add ${namespace}.${name} v${version} to repository "${repositoryName}".`,
+        t`Failed to add collection${
+          collections.length === 1 ? '' : 's'
+        } to repository "${repositoryName}".`,
```

Comment:
this is problematic when translating to languages which do not use -s for plurals

I guess the proper way with plurals would be...

`import { plural } from '@lingui/macro';`

```suggestion
    plural(collections.length, {
      one: 'Failed to add collection to repository "{repositoryName}".',
      other: 'Failed to add collections to repository "{repositoryName}".',
    }).replace('{repositoryName}', repositoryName)
```

(unless I'm missing a nice way to add interpolation to plurals)

Reply:
Forgot about this, this is much better better, thanks :+1:

Reply:
seems like `.replace('{repositoryName}', repositoryName),` doesn't replace variable in the string, but it works like this  
```
 plural(collections.length, {
    one: `Failed to add collection to repository "${repositoryName}".`,
    other: `Failed to add collections to repository "${repositoryName}".`,
})
```